cypress/e2e/bottomNavBar/careerTabTests.js

```diff
@@ -0,0 +1,44 @@
+import { onCareerPage } from "../../support/page_objects/careerPage.js"
+import {onMainPage} from "../../support/page_objects/mainPage.js"
+
+describe('Testing careera tab', () => {
+  beforeEach(() => {
+    onMainPage.visit()
+    onMainPage.navigateToCareeraTab()
+    onMainPage.acceptCookiesEn()
+  })
+
+  
+  it('Career tab should contain right titles', () => {
+    onCareerPage.checkTitles()
+  })
+
+  it('"\ Who are we looking for "\ section which doesn`t have any offers should contain right text', () => {
+    onCareerPage.whoAreWeLookingForSectionShouldContainRightText()
+  })
+
+  it('Check redirection to Polish offers',()=>{
+    onCareerPage.checkRedirectionToPlOffers()
+  })
+
+  it('"\What we offer"\ section should contain right text',()=>{
+    onCareerPage.whatWeOfferSectionShouldContainRightText()
+  })
+
+  it('"\Meet the HR team "\ section should contain right text',()=>{
+    onCareerPage.meetTheHrTeamShouldContainRightText()
+  })
+
+  it.only('Tiles should contain right text',()=>{
+    onCareerPage.youWorkAsYouPleaseTileShouldContainRightText()
```

Comment:
You have many methods, which are doing the same stuff for different elements.
You could create a single method which will work for different arguments